src/index.html

```diff
@@ -0,0 +1,4 @@
+<!DOCTYPE html>
+<body>
+    Hello, world!
+</body>
```

Comment:
Add an empty line at the end of your code

![30](https://user-images.githubusercontent.com/66570142/111797238-a7d18280-88d1-11eb-88fb-601f9b9141b8.png)